sdk/monitor/Azure.Monitor.Ingestion/src/LogsIngestionClient.cs

```diff
@@ -2,9 +2,12 @@
 // Licensed under the MIT License.
 
 using System;
+using System.Buffers;
```

Comment:
nit: remove this and then we should be able to avoid fully qualifiying the Azure.Core version throughout.